relationships/candidates/AWSELASTICACHEREDISCLUSTER.yml

```diff
@@ -0,0 +1,20 @@
+category: AWSELASTICACHEREDISCLUSTER
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSELASTICACHEREDISCLUSTER
```

Comment:
I could't find the tags defined here in any Redis cluster entity. These are tags for the Redis node entity: https://staging.onenr.io/0yw4Y17rgR3

Reply:
My bad, these tags are also present in cluster entities: https://staging.onenr.io/08jqWvx6Owl

Reply:
Since Redis nodes also have these tags, so we should include both entity types here.
I think that the case of an application connecting directly to a node endpoint is valid.